components/tabs/stories/template.js

```diff
@@ -16,11 +17,115 @@ export const Template = ({
   isQuiet,
   isEmphasized,
   isCompact,
-  items,
+	labelWithIcons,
+	iconOnly,
+	isOpen = true,
   selectorStyle = {},
   style = {},
+	overflow,
+	popoverOffset,
   ...globals
 }) => {
+
+	let displayedItems = []
+	let verticalSelectorStyle = {}
+
+	if (orientation === "vertical" && !isCompact) {
+		verticalSelectorStyle = {"block-size": "46px"}
+	} else if (orientation === "vertical" && isCompact) {
+		verticalSelectorStyle = {"block-size": "32px"}
+	} else {
+		verticalSelectorStyle = false
+	}
+
+	if (labelWithIcons) {
+		selectorStyle = verticalSelectorStyle ? verticalSelectorStyle : {"inline-size": "60px"},
+		displayedItems = [
+			{
+				id: "tab-1",
+				label: "Tab 1",
+				icon: "Folder",
+				isSelected: true
+			},
+			{
+				id: "tab-2",
+				label: "Tab 2",
+				icon: "Image"
+			},
+			{
+				id: "tab-3",
+				label: "Tab 3",
+				icon: "Document"
+			}
+		]
+	} else if (iconOnly){
+		selectorStyle = verticalSelectorStyle ? verticalSelectorStyle : {"inline-size": "20px"},
+		displayedItems = [
+			{
+				id: "tab-1",
+				icon: "Folder",
+				isSelected: true
+			},
+			{
+				id: "tab-2",
+				icon: "Image"
+			},
+			{
+				id: "tab-3",
+				icon: "Document"
+			}
+		]
+	} else {
+		selectorStyle = verticalSelectorStyle ? verticalSelectorStyle : {"inline-size": "35px"},
+		displayedItems = [
+			{
+				id: "tab-1",
+				label: "Tab 1",
+				isSelected: true
+			},
+			{
+				id: "tab-2",
+				label: "Tab 2",
+			},
+			{
+				id: "tab-3",
+				label: "Tab 3",
+			}
+		]
+	}
+
+	const overflowHtml = html`
```

Comment:
I couldn't tell if overflowHtml is used elsewhere or if it's just returned below?  If just returned below, can we skip adding it to a variable at all and just wrap this in the if statement and return the HTML directly?